go/cmd/goserver/server.go

```diff
@@ -0,0 +1,26 @@
+package main
```

Comment:
It's not clear to me why we call this folder `goserver`. Seems redundant. would `go/cmd/server/...` make more sense?